ow_lander/scripts/antenna_pan_tilt_action_client.py

```diff
@@ -19,18 +21,28 @@ def wrap_angle(angle):
         angle -= 2 * pi
     while angle < -(pi-tolerance):
         angle += 2 * pi
-    return angle 
+    return angle
+
 
 def antenna_client():
- 
-    client = actionlib.SimpleActionClient('AntennaPanTiltAction', ow_lander.msg.AntennaPanTiltAction)
 
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'pan', type=float, help='Antenna pan value in radians', nargs='?', default=0, const=0)
+    parser.add_argument(
+        'tilt', type=float, help='Antenna tilt value in radians', nargs='?', default=0, const=0)
+    args = parser.parse_args()
+    rospy.loginfo("Requetsed pan value: %s", args.pan)
+    rospy.loginfo("Requetsed tilt value: %s", args.tilt)
```

Comment:
[optional] You could save some lines and repetition by making a function that neatly prints each arguments' name and value just like you've done here.

```suggestion
    print_arguments(args)
```

The function would look like the following, and ideally you would import it from some shared utility function file since every `*_action_client.py` script would use it.
```Python
def print_arguments(args):
    # convert from namespace to dict and print pairs
    for name, value in zip(vars(args).keys(), vars(args).values()):
        rospy.loginfo("Requested %s value: %s" % (name, value))
```

Reply:
Thank you for this.. I have added this